### ISSUE TITLE
NVDA Feedback: Missing arrow key commands in test 7 through 10 in Radio Group Example Using aria-activedescendant, V25.02.12

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-at#1243]
## Description of Behavior

Same description as #1242.

## Test Setup

- Test File: [test-20-navToNextUncheckedRadio-nvda.collected.html](https://aria-at.netlify.app/tests/radiogroup-aria-activedescendant/test-20-navToNextUncheckedRadio-nvda.collected.html)
- Report Page: [Link](https://aria-at.w3.org/report/163635/targets/353#result-Yjc4YeyIxMiI6ODM5fQzhiNm)
- AT: NVDA (Version 2023.3)
- Browser: Chrome (Version 133)
